index.js

```diff
@@ -1,75 +1,75 @@
-const bodyParser = require('body-parser')
-const express = require('express')
-const logger = require('morgan')
-const app = express()
+const bodyParser = require('body-parser');
+const express = require('express');
+const logger = require('morgan');
+const app = express();
 const {
   fallbackHandler,
-  notFoundHandler,
   genericErrorHandler,
-  poweredByHandler
-} = require('./handlers.js')
+  poweredByHandler,
+} = require('./handlers.js');
 
 // For deployment to Heroku, the port needs to be set using ENV, so
 // we check for the port number in process.env
-app.set('port', (process.env.PORT || 9001))
+app.set('port', process.env.PORT || 9001);
 
-app.enable('verbose errors')
+app.enable('verbose errors');
 
-app.use(logger('dev'))
-app.use(bodyParser.json())
-app.use(poweredByHandler)
+app.use(logger('dev'));
+app.use(bodyParser.json());
+app.use(poweredByHandler);
 
 // --- SNAKE LOGIC GOES BELOW THIS LINE ---
 
 //  This function is called everytime your snake is entered into a game.
-//  cherrypy.request.json contains information about the game that's about to be played.
+//  request.body contains information about the game that's about to be played.
 // TODO: Use this function to decide how your snake is going to look on the board.
 app.post('/start', (request, response) => {
-  console.log("START");
+  console.log('START');
+  console.log(request.body);
 
   // Response data
   const data = {
-    color: '#888888',
-    headType: "regular",
-    tailType: "regular"
-  }
+    color: '#888888', // Choose a hex colour code to style your snake https://www.google.com/search?q=hex+colour+picker
+    headType: 'regular',
+    tailType: 'regular',
+  };
 
-  return response.json(data)
-})
+  return response.json(data);
+});
 
 // This function is called on every turn of a game. It's how your snake decides where to move.
 // Valid moves are "up", "down", "left", or "right".
-// TODO: Use the information in cherrypy.request.json to decide your next move.
+// TODO: Use the information in request.body to decide your next move.
 app.post('/move', (request, response) => {
-  var data = request.body;
+  console.log(request.body);
```

Comment:
Can you revert this back (or commend it out) - we specifically chose to not spam the logs with full request bodies.